packages/experimental/Avatar/src/JSAvatarTokens.windows.ts

```diff
@@ -0,0 +1,271 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { JSAvatarTokens } from '.';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+
+export const defaultJSAvatarTokens: TokenSettings<JSAvatarTokens, Theme> = (t: Theme) =>
```

Comment:
>defaultJSAvatarTokens [](http://example.com/codeflow?start=13&length=21)

Just FYI that we generally use the windows platform as the default, so we haven't been making .windows files for tokens

Reply:
I saw that in Button folder there's ButtonTokens.windows.ts. That's why I added it here. I'll remove this file